src/helpers.js

```diff
@@ -60,6 +60,9 @@ export function eachOperation(spec, cb, find) {
   // Iterate over the spec, collecting operations
   for (const pathName in paths) {
     for (const method in paths[pathName]) {
+      if (method.toUpperCase() === 'PARAMETERS') { 
```

Comment:
What is this check for? Would be great to add a comment clarifying the intention.